examples/experimental/dagster-airlift/dagster_airlift/core/load_defs.py

```diff
@@ -68,9 +67,8 @@ def build_defs_from_airflow_instance(
     return Definitions.merge(
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +62]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->might as well rename all the instances of `resolved_defs` to `mapped_defs` at this point<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/25054?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
Yea I'll do that in a followup I think